### ISSUE TITLE
Build completed with a result of 'Failed' 

### ISSUE THREAD

**Issue by GoncharovAnton** [googleads/googleads-mobile-unity#2993]
Hello. When adding Google Mobile Ads Unity Plugin v8.5.3 to a project I get the following error.

### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2023.1.16f
  * Google Mobile Ads Unity plugin version: 8.5.3
  * Platform: Android

### [REQUIRED] Step 2: Describe the problem

Modifying Android project files failed with output:
System.InvalidOperationException: Sequence contains no matching element
  at System.Linq.Enumerable.First[TSource] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] predicate) [0x00011] in <c9eddfa77f7c453c96c40d8d4011faf6>:0 
  at ModifyProjectScript.OnModifyAndroidProjectFiles (UnityEditor.Android.AndroidProjectFiles projectFiles) [0x001f1] in D:\UnityProjects\Angry cubes\Angry cubes\Assets\Editor\Android\ModifyProjectScript.cs:62 
  at UnityEditor.Android.PostProcessor.ProcessModifyProjectFilesCallbacks.ModifyAndroidProjectFiles (AndroidPlayerBuildProgram.Data.AndroidProjectFileArgs androidProjectFiles) [0x006fb] in <a695a5ade80b47c78dcc7b138c956499>:0 
  at UnityEditor.Android.AndroidBuildPostprocessor.ModifyAndroidProjectCallback (AndroidPlayerBuildProgram.Data.AndroidProjectFileArgs args) [0x00001] in <a695a5ade80b47c78dcc7b138c956499>:0 
  at (wrapper delegate-invoke) System.Action`1[AndroidPlayerBuildProgram.Data.AndroidProjectFileArgs].invoke_void_T(AndroidPlayerBuildProgram.Data.AndroidProjectFileArgs)
  at Bee.BeeDriver.BuildRequest+<>c__DisplayClass63_0`1[T].<RegisterRPCCallback>b__0 (System.Object o) [0x00000] in <2ddc5b9b369743eeb8936be4f2e8eaae>:0 
  at Bee.BeeDriver.BeeDriver_RunBackend+<>c__DisplayClass1_0.<ProcessRPCRequest>b__0 () [0x00021] in <2ddc5b9b369743eeb8936be4f2e8eaae>:0 
  at System.Threading.Tasks.Task.InnerInvoke () [0x0000f] in <07ba9f98d9794cf19aca0392570e5637>:0 
  at System.Threading.Tasks.Task.Execute () [0x00000] in <07ba9f98d9794cf19aca0392570e5637>:0 
--- End of stack trace from previous location where exception was thrown ---

  at Bee.BeeDriver.BeeDriver_RunBackend.ProcessRPCRequest (Bee.BinLog.RPCActionMessage msg, Bee.BeeDriver.BuildRequest+RPCCallback rpcCallback, IPCConnection ipcConnection, System.Threading.Tasks.Task writePipeConnectionTask, Bee.BeeDriver.InternalState state) [0x00102] in <2ddc5b9b369743eeb8936be4f2e8eaae>:0 
UnityEngine.GUIUtility:ProcessEvent (int,intptr,bool&)

**Comment by NVentimiglia** [comment]
Hi @GoncharovAnton The cause of the error `ModifyProjectScript` is not a Google script.

**Comment by GoncharovAnton** [comment]
Thank you for your response! There were no problems with the assembly. When adding a plugin I get an error. Do you have any ideas why this could be happening?

**Comment by NVentimiglia** [comment]
@GoncharovAnton 

Are you able to replicate this issue with a minimal sample, such as our hello world sample app?

I will need a stack trace which shows GoogleMobileAds as a source to move forward.

**Comment by GoncharovAnton** [comment]
You're right. I made a minimal sample and there were no errors. The problem was in another plugin that included the ModifyProjectScript.cs script. Thanks for support!

**Comment by NVentimiglia** [comment]
I have reopened this issue. I have replicated the issue by installing firebase.

**Comment by NVentimiglia** [comment]
@GoncharovAnton  I have a work around, be sure to setup unity to use [gradle resolution](https://developers.google.com/admob/unity/quick-start#include_external_dependencies) and use the Android Studio Export option when building.

**Comment by NVentimiglia** [comment]
Also note Firebase requires Gradle 7.8 minversion. You may need to update Unity to 2022.3 or upgrade your [gradle](https://developers.google.com/admob/unity/gradle)
